CHANGELOG.md

```diff
@@ -1,5 +1,79 @@
 # Changelog
 
+## [0.58.0]
+
+- A lot of community effort in [modernizing](https://github.com/facebook/react-native/issues/21581) and [stengthening flow types](https://github.com/facebook/react-native/issues/22100) for core components. Thank you [@TheSavior](https://github.com/TheSavior) for leading these efforts, and thank you to all of the community members who helped get this done.
+- There was great continued progress towards the new [Fabric re-architecture](https://github.com/react-native-community/discussions-and-proposals/issues/4) this month
```

Comment:
Also, how worthwhile is this to include anyways? We would essentially be saying this every release until we ship, right?

Reply:
Yeah, this was left open intentionally. I think it's worthwhile to signal to folks how progrees was, but this statement doesn't satisfy that goal. Consider this entire top matter as needing serious refinement, and I don't imagine that Fabric will be mentioned.

Reply:
is pard of the fabric re-architecture in 0.58? if not this shouldn't be there. But if there is foundational work to support it that is being included in this release then I think this could be reworded a bit and be reasonable to stay

Reply:
Based on conversations this week around fabric, I'm going to remove this for now. It's still good progress, but I think it's misleading to report on it without some expectation on when it'll be available. I don't have that information available to share today, so this is a bit of a useless statement today. Thanks for the feedback @TheSavior  and @raphaeleidus